### DISCUSSION TITLE
Permissions setup with LuckPerms

### DISCUSSION THREAD

**Discussion by DedMaxim** [FTBTeam/FTB-Mods-Issues#1110] [Q&A]
Hi. So, I want to set different perks to different groups using LuckPerms. For example, I want to make it so group "vip" has 3 homes instead of 1. FTB documentation says there is a permission for it:

ftbessentials.home.max - [integer] How many different homes you can create

However, it doesn't seem to work if I set it to ftbessentials.home.max.3
I also tried doing some different stuff like:
ftbessentials.home.max 3
ftbessentials.home.max-3
ftbessentials.home.max - 3
ftbessentials.home.max: 3

None of them affected. So, how can I do this then? And is it possible at all outside FTB Ranks?

**Comment by bombitmanbomb** [comment]
I believe FTB uses meta tags for those. so `meta.ftbessentials\.home\.max.3` (meta.key.value)

**Reply by botagas** [reply]
How about ftbchunks?
I've set the following with no effect:
```
meta.ftbchunks\.max_claimed.1000
meta.ftbchunks\.max_force_loaded.50
```
Yet, nothing changes.

**Reply by mmouradev** [reply]
@botagas Did you succeed?

**Reply by botagas** [reply]
> @botagas Did you succeed?

I believe I did, but I do not remember how at the moment. I could probably check later today.
